test/testcafe/spec/Identify_spec.js

```diff
@@ -140,7 +141,7 @@ test.meta('gen3', false).requestHooks(identifyRequestLogger, identifyMock)('shou
   await t.expect(req.method).eql('post');
   await t.expect(req.url).eql('http://localhost:3000/idp/idx/identify');
   await t.expect(reqHeaders['x-device-fingerprint']).notOk(); // only enabled if `features.deviceFingerprinting` is true
-  await t.expect(reqHeaders['x-okta-user-agent-extended']).contains(` okta-signin-widget-${config.version}`);
+  await t.expect(reqHeaders['x-okta-user-agent-extended']).contains(` okta-signin-widget-g3-${config.version}`);
```

Comment:
Does this need to be different between gen2 and gen3? You may need to conditionally assert a different value or use a regex instead (i think a regex would be more elegant)

Reply:
yeah I forgot it def needs to be different. ill try with a regex